src/sql/store.py

```diff
@@ -58,11 +58,10 @@ def render(self, query, with_=None):
     @modify_exceptions
     def store(self, key, query, with_=None):
         if "-" in key:
-            print(
-                "Dash is not suggested in standard sql clause,"
-                " please replace this with an underscore, e.g. no-nulls -> no_nulls"
+            raise SyntaxError(
```

Comment:
this should be a UsageError (`from IPython.core.error import UsageError`)

Reply:
Fix in [d50f586](https://github.com/ploomber/jupysql/pull/164/commits/d50f58667efbc07e8579ee735b6c0e4c7094c596)

Resolved conflict